### ISSUE TITLE
Bug: Settings not persisted on UI

### ISSUE THREAD

**Issue by Olaleye-Blessing** [open-sauced/insights#1456]
### Describe the bug

The checkbox for local time is not persisted. It's persisted briefly when the page is still loading.

https://github.com/open-sauced/insights/assets/70102539/56676279-47d7-4a22-b558-3552ed0dd5ce

### Steps to reproduce

- Go to user's settings page, [https://insights.opensauced.pizza/user/settings](https://insights.opensauced.pizza/user/settings).
- Check the `Display current local time on profile` and refresh the page.
- Notice the bug.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by t-bello7** [comment]
.take

**Comment by jpmcb** [comment]
This isn't a bug: you need to select "Update profile" to actually save those settings and persist them to the Open-Sauced API. Closing this issue since there's not action to take but happy to continue the conversation here.

@Olaleye-Blessing - I'm curious if this was confusing for you or if you have a suggestion how to make this more clear?

**Comment by Olaleye-Blessing** [comment]
Yes, the change is persisted to the API but not persisted on the UI after reloading the page.

Got to the page, make sure it the checkbox is checked, updated profile, then refresh the page.
